.github/workflows/ci.yml

```
@@ -0,0 +1,17 @@
+name: ci
+on: push
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
```

I recommend instead creating a `.nvmrc` file containing the version (e.g., https://github.com/coveo/ui-kit/blob/9705d0bd14d9da29e5deca3a80290f1cc9fe6252/.nvmrc) and then using it from the action like so:
```suggestion
          node-version-file: '.nvmrc'
```